support/oidc-discovery-provider/config.go

```diff
@@ -130,11 +134,16 @@ func ParseConfig(hclConfig string) (_ *Config, err error) {
 
 	switch {
 	case c.ACME == nil:
-		if c.InsecureAddr == "" {
+		if c.InsecureAddr == "" && c.ListenSocketPath == "" {
 			return nil, errs.New("acme section must be configured")
```

Comment:
"either acme or listen_socket_path must be configured"

Reply:
Updated